proposals/threads/Overview.md

```diff
@@ -389,7 +389,7 @@ For the web embedding, `i32.atomic.wait` is equivalent in behavior to executing
 1. Let `memory` be a `WebAssembly.Memory` object for this module.
 1. Let `buffer` be `memory`([`Get`][](`memory`, `"buffer"`)).
 1. Let `int32array` be [`Int32Array`][](`buffer`).
-1. Let `result` be [`Atomics.wait`][](`int32array`, `address`, `expected`, `timeout`),
+1. Let `result` be [`Atomics.wait`][](`int32array`, `address`, `expected`, `timeout` / 1000000),
```

Comment:
For readabilty I think it's better to use scientific notation  `1x10^6`?

Reply:
Added here: https://github.com/WebAssembly/threads/pull/111